corehq/apps/app_manager/models.py

```diff
@@ -994,9 +994,113 @@ def validate(self, value, required=True):
 
 
 class NavMenuItemMediaMixin(DocumentSchema):
+    """
+        Language-specific icon and audio.
+        Properties are map of lang-code to filepath
+    """
+    media_image = SchemaDictProperty(JRResourceProperty)
+    media_audio = SchemaDictProperty(JRResourceProperty)
+
+    @classmethod
+    def wrap(cls, data):
+        # ToDo - Remove after migration
+        for media_attr in ('media_image', 'media_audio'):
+            old_media = data.get(media_attr, None)
+            if old_media and isinstance(old_media, basestring):
+                new_media = {'default': old_media}
+                data[media_attr] = new_media
+
+        return super(NavMenuItemMediaMixin, cls).wrap(data)
+
+    def _get_media_by_language(self, media_attr, lang, strict=False):
+        # if strict is False, media for a random lang is returned if given lang
```

Comment:
Update comment to be a doc string. Also its not a 'random' lang, its the first one sorted by lang.